src/extendedstorage.cpp

```diff
@@ -568,9 +557,9 @@ bool ExtendedStorage::validateNotebooks()
 
 bool ExtendedStorage::isValidNotebook(const QString &notebookUid)
 {
-    Notebook::Ptr nb = notebook(notebookUid);
-    if (!nb.isNull()) {
-        if (nb->isRunTimeOnly() || nb->isReadOnly()) {
+    const Notebook &nb = notebook(notebookUid);
+    if (nb.isValid()) {
```

Comment:
This is an interesting method. Could assume it's the same as notebook(uid).isValid() but it's not exactly. Three cases using it, pondering if it's needed or should the name be adjusted a little, e.g. notebookWritable() or something to that direction.